src/sentry/rules/filters/issue_type.py

```diff
@@ -0,0 +1,46 @@
+from collections import OrderedDict
+from typing import Any, Union
+
+from django import forms
+
+from sentry.eventstore.models import Event, GroupEvent
+from sentry.rules import EventState
+from sentry.rules.filters import EventFilter
+from sentry.types.issues import GroupType
+
+TYPE_DESCRIPTIONS = {
+    GroupType.ERROR.value: "Errors",
+    GroupType.PERFORMANCE_N_PLUS_ONE.value: "Performance (N+1)",
+    GroupType.PERFORMANCE_SLOW_SPAN.value: "Performance (slow span)",
+    GroupType.PERFORMANCE_SEQUENTIAL_SLOW_SPANS.value: "Performance (sequential slow spans)",
+    GroupType.PERFORMANCE_LONG_TASK_SPANS.value: "Performance (long task spans)",
+    GroupType.PERFORMANCE_RENDER_BLOCKING_ASSET_SPAN.value: "Performance (render blocking asset span)",
+    GroupType.PERFORMANCE_DUPLICATE_SPANS.value: "Performance (duplicate spans)",
```

Comment:
we can omit these btw. we don't create perf issues out of them yet and we only surface n+1 db query detector to users for now